src/main/java/medic/gateway/alert/WebappPoller.java

```diff
@@ -61,41 +65,44 @@ public SimpleResponse pollWebapp() throws JSONException, MalformedURLException {
 		return response;
 	}
 
-	public Boolean moreMessagesToSend() {
```

Comment:
This function has the new change, the rest in this file is about fixing spaces and parentheses. 

It'll stop polling from webapp if there's an error. Wakefull will poll again automatically in 1 min and send any pending item. Previously discussed [here](https://github.com/medic/medic-gateway/pull/161/files#r468959249)